misk/src/main/kotlin/misk/time/ClockModule.kt

```diff
@@ -2,9 +2,12 @@ package misk.time
 
 import misk.inject.KAbstractModule
 import java.time.Clock
+import java.time.ZoneOffset
+import java.util.*
 
 class ClockModule : KAbstractModule() {
   override fun configure() {
     bind<Clock>().toInstance(Clock.systemUTC())
+    TimeZone.setDefault(TimeZone.getTimeZone(ZoneOffset.UTC))
```

Comment:
I think we gotta move this somewhere else. Not urgent, but this is not how Guice wants to work
https://github.com/google/guice/wiki/ModulesShouldBeFastAndSideEffectFree

Reply:
Ah, whoops. I will fix in a follow-up.